src/custom-types.json

```diff
@@ -142,7 +142,12 @@
     ]
   },
   "application/tar": {
-    "compressible": true
+    "compressible": true,
+    "extensions": ["tar","tar.gz","taz","tgz"],
+    "sources": [
+      "https://www.gnu.org/software/tar/",
```

Comment:
I cannot find where on this page any of these extensions are listed as being for this mime type. If you can point it out, I maye have overlooked.

Reply:
Is this page link sufficient enough? I can change to this if it is.
https://www.gnu.org/software/tar/manual/html_node/gzip.html

The page contains a list of extensions like this:
```
Suffix	Compression program
‘.gz’	gzip
‘.tgz’	gzip
‘.taz’	gzip
‘.Z’	compress
‘.taZ’	compress
‘.bz2’	bzip2
‘.tz2’	bzip2
‘.tbz2’	bzip2
‘.tbz’	bzip2
‘.lz’	lzip
‘.lzma’	lzma
‘.tlz’	lzma
‘.lzo’	lzop
‘.xz’	xz
‘.zst’	zstd
‘.tzst’	zstd
```

Reply:
The page needs to say that a given extension is identified by a given mime type. Is there a page that procides that or a page with a table of those archive formats to their mime type?

Reply:
Can't find such a page unfortunately. There is no official definition on the ".tgz" extension. This is a discussion that might be helpful on coming up with a mime type for the extension:
https://superuser.com/questions/901962/what-is-the-correct-mime-type-for-a-tar-gz-file

Can we define a type for it which we think appropriate for now? And then if there is an official definition, we can switch to it? I think either of these two types is acceptable for me:
- application/tar
- application/tar+gzip

I'm open to other ideas as well.

Reply:
Thia module is just an aggregatation of the three sources listed at the top of the readme. As a curiosity if none of those three sources have the mapping but there is a clear, official source, we can add it here. Absent that, we wouldn't add it.

Reply:
Just to double check that I understand your comment. Are you saying that if we don't provide a clear and official web page, you will not approve this Pull Request?

Reply:
The requirements are listed at https://github.com/jshttp/mime-db#contributing

Reply:
Thank you! I will try to follow the requirements and see if we get this done.
In the meantime, closing this PR. Will re-open it if needed.